.classpath

```diff
@@ -2,7 +2,11 @@
 <classpath>
 	<classpathentry kind="src" path="src"/>
 	<classpathentry excluding="**/package.html" kind="src" path="test"/>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/jdk1.8.0_321">
```

Comment:
It's not clear to me why this change in `.classpath` is checked in.  It appears that the change switches from a generic container to a very specific java version (1.8.0_321).  I suspect that this might cause problems for developers using a slightly different version of java.  Can someone at least explain the purpose of this change?

Reply:
Snuck in by mistake.  I can not build using the standard JRE, I need to use the one I installed.  It should not have been pushed.   Will fix.

Reply:
This has not been fixed yet.

Reply:
Just pushed fix in [this commit.](https://github.com/pc2ccs/pc2v9/pull/732/commits/a344d420f9cce047ee0006197035690a7bee7500)